ui/components/ui/typography/typography.js

```diff
@@ -13,24 +13,92 @@ import Box, { MultipleSizes } from '../box';
 
 const { H6, H7, H8, H9 } = TYPOGRAPHY;
 
+export const ValidColors = [
+  COLORS.TEXT_DEFAULT,
+  COLORS.TEXT_ALTERNATIVE,
+  COLORS.TEXT_MUTED,
+  COLORS.OVERLAY_INVERSE,
+  COLORS.PRIMARY_DEFAULT,
+  COLORS.PRIMARY_INVERSE,
+  COLORS.SECONDARY_DEFAULT,
+  COLORS.SECONDARY_INVERSE,
+  COLORS.ERROR_DEFAULT,
+  COLORS.ERROR_INVERSE,
+  COLORS.SUCCESS_DEFAULT,
+  COLORS.SUCCESS_INVERSE,
+  COLORS.WARNING_INVERSE,
+  COLORS.INFO_DEFAULT,
+  COLORS.INFO_INVERSE,
+  /**
+   * COLORS BELOW HAVE BEEN DEPRECATED
+   */
+  COLORS.UI1,
+  COLORS.UI2,
+  COLORS.UI3,
+  COLORS.UI4,
+  COLORS.BLACK,
+  COLORS.GREY,
+  COLORS.NEUTRAL_GREY,
+  COLORS.WHITE,
+  COLORS.PRIMARY1,
+  COLORS.PRIMARY2,
+  COLORS.PRIMARY3,
+  COLORS.SECONDARY1,
+  COLORS.SECONDARY2,
+  COLORS.SECONDARY3,
+  COLORS.SUCCESS1,
+  COLORS.SUCCESS2,
+  COLORS.SUCCESS3,
+  COLORS.ERROR1,
+  COLORS.ERROR2,
+  COLORS.ERROR3,
+  COLORS.ALERT1,
+  COLORS.ALERT2,
+  COLORS.ALERT3,
+];
+
+export const ValidTags = [
+  'dd',
+  'div',
+  'dt',
+  'em',
+  'h1',
+  'h2',
+  'h3',
+  'h4',
+  'h5',
+  'h6',
+  'li',
+  'p',
+  'span',
+  'strong',
+];
+
 export default function Typography({
   variant = TYPOGRAPHY.Paragraph,
-  className,
-  color = COLORS.BLACK,
-  tag,
-  children,
+  color = COLORS.TEXT_DEFAULT,
   fontWeight = 'normal',
   fontStyle = 'normal',
   align,
   overflowWrap,
-  boxProps = {},
+  tag,
   margin = [1, 0],
+  boxProps = {},
+  className,
+  children,
 }) {
+  let Tag = tag ?? variant;
+  let strongTagFontWeight;
+
+  if (Tag === 'strong') {
+    strongTagFontWeight = FONT_WEIGHT.BOLD;
```

Comment:
@brad-decker I noticed that the `<strong>` tag doesn't actually make the font bold so have added it here. Oh actually it might not be working because I removed the other font bolding options in `design-system.js` I'll check.

Reply:
Ok I reverted some changes and <strong> tag still shows up as a normal font-weight instead of bold